_posts/2020-02-10-waku-current-state.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+name:  "Waku current state"
+title:  "Waku current state"
+date:   2020-02-10 12:00:00 +0800
+author: oskarth
+published: false
+permalink: /waku-current-state
+categories: research
+summary: A research log. Waku current state.
+image: /assets/img/TODOFIXME.png
+discuss: https://forum.vac.dev/t/TODOFIXME
+---
+
+TODO: Picture for bottleneck or Waku
+
+# Where we are at with Waku?
+
+Waku is our fork of Whisper where we address the shortcomings of Whisper in an iterative manner. We've seen in previous post that Whisper doesn't scale, and why. Here's an update on where we are at with Waku since that [last post](https://vac.dev/fixing-whisper-with-waku).
```

Comment:
```suggestion
Waku is our fork of Whisper where we address the shortcomings of Whisper in an iterative manner. We've seen a in [previous post](https://vac.dev/fixing-whisper-with-waku) that Whisper doesn't scale, and why. Here's an update on where we are at with Waku since then.
```